util/rate_limiter_test.cc

```diff
@@ -170,13 +177,19 @@ TEST_F(RateLimiterTest, GeneratePriorityIterationOrder) {
       {Env::IO_USER, Env::IO_LOW, Env::IO_MID, Env::IO_HIGH}};
 
   for (int i = 0; i < 4; ++i) {
+    // These are variables for making sure the following callbacks are called
+    // and the assertion in the last callback is indeed excuted
+    bool high_pri_iterated_after_mid_low_pri_set_ = false;
+    bool mid_pri_itereated_after_low_pri_set_ = false;
+    bool pri_iteration_order_verified_ = false;
```

Comment:
Usually trailing underscore indicates a member variable.

Reply:
Will fix it!